dspace/config/crosswalks/oai/transformers/driver-commons.xsl

```diff
@@ -46,7 +46,7 @@
 		</xsl:variable>
 
 		<xsl:choose>
-			<xsl:when test="count($bitstreams) = 0 or count($embargoed[text() = 'forever']) = count($bitstreams) ">
+			<xsl:when test="not(count($bitstreams) = 0) and count($embargoed[text() = 'forever']) = count($bitstreams) ">
```

Comment:
en vez de `not(count($bitstreams) = 0) ` quizás quedaría mejor `count($bitstreams) > 0`

Reply:
copied